src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Controllers/WeatherForecastController.cs

```diff
@@ -1,13 +1,23 @@
-﻿using ComponentsWebAssembly_CSharp.Shared;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 #if (!NoAuth)
 using Microsoft.AspNetCore.Authorization;
 #endif
+#if (GenerateApi)
+using Microsoft.Extensions.Configuration;
+using Microsoft.Identity.Web;
+using System.Net;
+using System.Net.Http;
+#endif
+#if (GenerateGraph)
+using Microsoft.Graph;
+#endif
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Identity.Web.Resource;
```

Comment:
This looks like it should only be needed when IndividualB2c or OrgAuth is specified right?

Reply:
Yes, you are right!